### PR TITLE
v2v: add a new case for weak denpendency of libguestfs-xfs

### PR DESCRIPTION
**Pull request by xiaodwan**
Add a new case for weak dependency of libguestfs-xfs.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.cmd_free.weak_dependency.input_mode.none.output_mode.none: PASS (6.46 s)

File: v2v/tests/src/v2v_options.py [hunk 1]
```diff
@@ -886,6 +886,8 @@ def check_alloc():
     finally:
         if hypervisor == "esx":
             process.run("rm -rf %s" % vpx_passwd_file)
+        if checkpoint == "weak_dendency":
+            process.run("yum install libguestfs-xfs -y", ignore_status=True)
```
Comment (chunfuwen) [inline v2v/tests/src/v2v_options.py:890]: we usually add shell=True when call process.run to avoid special characters issue

Comment (xiaodwan) [reply v2v/tests/src/v2v_options.py]: @chunfuwen I changed it to utils_package.package_install function.

**Comment by xiaodwan** [comment]
Merged this PR since the change request were modified.

**Pull request merged**